kernel-scriptlets/rpm-script

```diff
@@ -330,6 +343,14 @@ case $op in
 			 [ "$(readlink /boot/initrd)" = "initrd-$kernelrelease-$flavor" ]; }; then
 		rm -f /boot/initrd "/boot/$image"
 	    fi
+	    if [ ! -f /boot/initrd ] || [ ! -f "/boot/$image"; then
```

Comment:
If I'm not mistaken you have a syntax error here, missing `]`

Reply:
ups